src/source/PeerConnection/PeerConnection.c

```diff
@@ -863,6 +864,9 @@ STATUS initKvsWebRtc(VOID)
     UNUSED_PARAM(SSL_library_init());
 
     CHK_STATUS(initSctpSession());
+
```

Comment:
Add a check in the beginning of this function before SRAND to make this idempotent. Simply

CHK(ATOMIC_LOAD_BOOL(&gKvsWebRtcInitialized) != TRUE, retStatus);